serialization/serializer_test.go

```diff
@@ -6,10 +6,11 @@ import (
 	"context"
 	"fmt"
 	"math/rand"
-	"sync/atomic"
 	"testing"
 	"time"
 
+	"go.uber.org/atomic"
+
```

Comment:
Newline here looks odd? Is that a linter thing?

Reply:
Yup.